astropy/io/fits/hdu/hdulist.py

```diff
@@ -51,6 +51,7 @@ def fitsopen(
     cache=True,
     lazy_load_hdus=None,
     ignore_missing_simple=False,
+    load_variable_length_data=True,
```

Comment:
Do we need to expose this ? In a previous version only the class variable was used, which I think is fine. 
But maybe this is a rebase issue, since it doesn't seem to be passed to `_readfrom` ?

Reply:
Oops yes rebase issue, will remove

Reply:
Fixed